guide/src/format/summary.md

```diff
@@ -34,11 +34,14 @@ allow for easy parsing. Let's see how you should format your `SUMMARY.md` file.
    ```markdown
    # Title of Part
 
-   - [Title of the Chapter](relative/path/to/markdown.md)
+   - [Title of the first  Chapter](relative/path/to/markdown.md)
+   - [Title of the second Chapter](relative/path/to/markdown2.md)
+      * [Title of a sub Chapter](relative/path/to/markdown3.md)
```

Comment:
I would recommend keeping the same style for nested lists, just to avoid any confusion about whether or not that is required. Most markdown styles that I have seen use only one kind of bullet.

```suggestion
      - [Title of a sub Chapter](relative/path/to/markdown3.md)
```